rosplan_planning_system/include/rosplan_planning_system/PlanDispatch/EsterelPlanDispatcher.h

```diff
@@ -11,7 +11,8 @@
 #include "PlanDispatcher.h"
 
 #include "rosplan_knowledge_msgs/KnowledgeItem.h"
-#include "rosplan_dispatch_msgs/EsterelPlan.h"
+#include "rosplan_dispatch_msgs/EsterelPlanEdge.h"
+#include "rosplan_planning_system/PlanParsing/PDDLEsterelPlanParser.h"
```

Comment:
This include should not be here. As far as I can tell the dispatcher makes no reference to the parser, so it will still be able to compile without it.
You might also be missing a reference to:
rosplan_dispatch_msgs/EsterelPlan.h
  and/or
rosplan_dispatch_msgs/EsterelPlanNode.h
However only add them if they are required.

Reply:
The include #include "rosplan_planning_system/PlanParsing/PDDLEsterelPlanParser.h" is still there (should not be)